test/docker/Dockerfile.testhost

```diff
@@ -1,5 +1,3 @@
-FROM ubuntu:focal
-
```

Comment:
It looks like this change was unintended.

Reply:
Done

Reply:
Thanks. Sorry to nitpick, but there are now a bunch of superfluous white space changes. Could you remove those? The dockerfiles have an extra space at the end of the the last line in the dependencies and this file shouldn't show in the diff at all.

Reply:
Oh, I see you added a second commit. It will be easier to notice those whitespace problems if you squash the commits together.